package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "colorimetry",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "https://github.com/publiclab/colorimetry.git",
+  "author": "jywarren",
+  "license": "GNU",
+  "dependencies": {
+    "base64-img": "^1.0.4",
+    "image-sequencer": "^3.1.1"
+  },
+  "scripts": {
+    "start": "node colorimetry.js"
```

Comment:
Let's put everything inside a `src/` directory and add an index.js file at the start. What do you say?